src/main/java/com/theateam/waaprojectspringbackend/controller/CustomersController.java

```diff
@@ -53,15 +60,45 @@ public void updateOffer(@PathVariable Long offerId, @RequestBody UpdateOfferDto
     }
 
     @GetMapping("/saved-properties")
-    public String getSavedProperties() {
-        return "Greetings from Customers GET saved-properties!";
+    public ResponseEntity<List<String>> getSavedProperties() {
```

Comment:
This should all go into service method. like `PropertyService.getSavedProperties(userId)`